libr/core/panels.c

```diff
@@ -57,6 +57,12 @@ static const char *menus_ReOpen[] = {
 };
 static const int reopenNum = ((int)sizeof (menus_ReOpen) / (int)sizeof (const char *)) - 1;
 
+static const char *menus_loadLayout[] = {
+	"Saved", "Default",
+	NULL
+};
+static const int loadLayoutNum = ((int)sizeof (menus_loadLayout) / (int)sizeof (const char *)) - 1;
```

Comment:
There are a lot of constructs like this, maybe it worth to make them a macro?

Reply:
change it here, we'll try to reduce the amount of commits fixing things from previous commits, that will make release notes easier